src/pages/workspaces/workspace/analysis/file-utils.test.ts

```diff
@@ -0,0 +1,103 @@
+import '@testing-library/jest-dom'
+
+import { act, renderHook } from '@testing-library/react-hooks'
+import { AzureStorage, AzureStorageContract } from 'src/libs/ajax/AzureStorage'
+import { GoogleStorage, GoogleStorageContract } from 'src/libs/ajax/GoogleStorage'
+import { workspaceStore } from 'src/libs/state'
+import {
+  CloudProviderType,
+  cloudProviderTypes
+} from 'src/libs/workspace-utils'
+import { defaultAzureWorkspace, defaultGoogleWorkspace } from 'src/pages/workspaces/workspace/analysis/_testData/testData'
+import {
+  AbsolutePath,
+  AnalysisFile,
+  getDisplayName,
+  getExtension,
+  getFileName,
+  useAnalysisFiles
+} from 'src/pages/workspaces/workspace/analysis/file-utils'
+import { getToolFromFileExtension, ToolLabel } from 'src/pages/workspaces/workspace/analysis/tool-utils'
+import { asMockedFn } from 'src/testing/test-utils'
+
+
+jest.mock('src/libs/ajax/GoogleStorage')
+jest.mock('src/libs/ajax/AzureStorage')
+
+//TODO: test commons
+const getTestFile = (abs: AbsolutePath, cloudProvider: CloudProviderType = cloudProviderTypes.GCP): AnalysisFile => ({
+  name: abs,
+  ext: getExtension(abs),
+  displayName: getDisplayName(abs),
+  fileName: getFileName(abs),
+  tool: getToolFromFileExtension(getExtension(abs)) as ToolLabel,
+  lastModified: new Date().getTime(),
+  cloudProvider
+})
+
+describe('file-utils', () => {
+  beforeEach(() => {
+    workspaceStore.reset()
+    const googleStorageMock: Partial<GoogleStorageContract> = ({
+      listAnalyses: jest.fn(() => Promise.resolve([]))
+    })
+    asMockedFn(GoogleStorage).mockImplementation(() => googleStorageMock as GoogleStorageContract)
+
+    const azureStorageMock: Partial<AzureStorageContract> = ({
+      listNotebooks: jest.fn(() => Promise.resolve([]))
+    })
+    asMockedFn(AzureStorage).mockImplementation(() => azureStorageMock as AzureStorageContract)
+  })
+
+  it('loads files from the correct cloud provider storage with a google workspace', async () => {
+    // Arrange
+    const fileList: AnalysisFile[] = [getTestFile('test/file1.ipynb' as AbsolutePath), getTestFile('test/file2.ipynb' as AbsolutePath)]
+
+    const calledMock = jest.fn(() => Promise.resolve(fileList))
```

Comment:
Suggest naming mocks based on the function they mock.
```suggestion
    const listAnalyses = jest.fn(() => Promise.resolve(fileList))
```